geomesa-utils/src/test/scala/org/locationtech/geomesa/utils/stats/StatTestHelper.scala

```diff
@@ -12,29 +12,31 @@ import org.geotools.feature.simple.SimpleFeatureBuilder
 import org.locationtech.geomesa.utils.geotools.SimpleFeatureTypes
 
 trait StatTestHelper {
-  val sftSpec = "strAttr:String,intAttr:Integer,longAttr:Long,doubleAttr:Double,floatAttr:Float,geom:Geometry:srid=4326,dtg:Date"
+  val sftSpec = "strAttr:String,intAttr:Integer,longAttr:Long,doubleAttr:Double,floatAttr:Float,cat1:Integer,cat2:String,geom:Geometry:srid=4326,dtg:Date"
```

Comment:
if you put cat1 and cat2 at the end of the list of attributes, then you probably won't need to modify the other unit tests